cheatSheets.md

```diff
@@ -47,6 +47,11 @@
 - [Flutter Cheat Sheet](https://howtodothisinflutter.com/): Basic flutter commands and sample code
 
 - [HTML Cheat Sheet](https://html.com/wp-content/uploads/html-cheat-sheet.pdf): This clear, easy-to-access PDF thoroughly explains HTML elements, with understandable explanations and examples.
+- 
```

Comment:
Please remove this unnecessary bullet point @g-paras

Reply:
Done 👍🏻